rasa/nlu/test.py

```diff
@@ -62,7 +63,7 @@
 
 ResponseSelectionEvaluationResult = namedtuple(
     "ResponseSelectionEvaluationResult",
-    "intent_target " "response_target " "response_prediction " "message " "confidence",
+    "intent_target response_key response_target response_prediction_full_intent response_prediction message confidence",
```

Comment:
It looks like with this change we can also get rid of `intent_target`, `response_target` and `response_prediction`. Can you scrub those off as well? There will be some more places where you will have to make changes.